tiledb/tests/test_dask.py

```diff
@@ -1,28 +1,17 @@
-try:
-    import dask, dask.array as da
+import pytest
 
-    import_failed = False
-except ImportError:
-    import_failed = True
-
-import os
+da = pytest.importorskip("dask.array")
 
+import sys
 import tiledb
 from tiledb.tests.common import DiskTestCase
 
 import numpy as np
 from numpy.testing import assert_array_equal, assert_approx_equal
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="does not run on windows")
```

Comment:
If we're standardizing on `sys.platform`, there are a few leftovers of `platform.system()` and `os.name` you may want to port.